code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -90,6 +90,16 @@
 	else
 		return ..()
 
+
+/mob/living/simple_animal/hostile/retaliate/goat/AttackingTarget()
+	. = ..()
+	if(. && ishuman(target))
+		var/mob/living/carbon/human/H = target
+		if(istype(H.dna.species, /datum/species/pod))
+			var/obj/item/bodypart/NB = pick(H.bodyparts)

```

Comment:
And yet you still make the same mistake.

Reply:
what are you referring to @RemieRichards

Reply:
I'm guessing empty list pick

Reply:
A mob should always have at least a chest though, something else has already terribly wrong if this runtimes

Reply:
And I've seen mobs ingame that are somehow missing their chest. I mean, SURE, it shouldn't happen, but it does.

Reply:
Isnt that just covering up the issue though?

Reply:
Well yes, but nobody seems to have cared enough to figure out the cause and fix

Reply:
#19398 olddddddd